portal/src/index.js

```diff
@@ -63,13 +62,6 @@ async function createApp(ssrContext) {
 
   pluginRouting(app.context)
 
-  const axiosInstance = axios(app.context)
-
-  // TODO: This should be replaced by modules (import)
-  store.$axios = axiosInstance
-  app.$axios = axiosInstance
-  Vue.prototype.$axios = axiosInstance
```

Comment:
There is actually merit here. It makes it really easy to mock $axios. Have you thought about that and what do you propose as an alternative if we decide we do want to test a complex frontend component?

An alternative I can suggest is to wrap axios in an injector.constant call. That way it becomes available with injector.get and is mockable again. Would be convention to use $axios for that.

I think we should have a strategy to mock axios, because if we decide to write some tests we don't want to first change the entire codebase again, before we can. This is the time to make a decision like this.

Reply:
Why can't we mock an import? It also returns the same instance, so it should be just as easy.

Reply:
What kind of tools do you use for mocking imports?